uwsift/view/scene_graph.py

```diff
@@ -87,6 +87,13 @@
 DEFAULT_TEXTURE_SHAPE = (4, 16)
 
 
+class CustomImage(Image):
+    """Custom image override to fix issue #404."""
+
+    def _init_texture(self, data, texture_format, **texture_kwargs):
+        return super()._init_texture(data, texture_format, internalformat="r32f")
```

Comment:
maybe it's more futureproof if we still pass on eventual texture_kwargs as in below, if you agree?
```suggestion
        return super()._init_texture(data, texture_format, internalformat="r32f", **texture_kwargs)
```